ui/core/components/pickers/number_picker.ts

```diff
@@ -8,6 +9,7 @@ export interface NumberPickerConfig<ModObject> extends InputConfig<ModObject, nu
 	id: string;
 	// Whether the picker represents a float value. Default `false`
 	float?: boolean;
+	fractionCount?: number;
```

Comment:
Minor nit, but this name confused me until I saw what it meant. Renaming to `maxDecimalDigits` or `decimalPrecision` would be clearer imo.